### DISCUSSION TITLE
Show collections on homepage

### DISCUSSION THREAD

**Discussion by thomseddon** [outline/outline#1509] [General]
Currently the "home" page shows a list of the most recently edited documents, which I think is a little confusing for new users.

Would you be open to a PR that added a "collections" tab to the home page?

Also, would you be open to a PR that allowed you to set the default tab for the home page and for collections (I'd prefer alphabetical).

Both of these suggestions are based on the idea that the primary use for a wiki is "reading" - information logically and predictably laid out. The current defaults feels like it's primarily geared towards "editing".

Let me know what you think!

**Comment by tommoor** [comment]
> Would you be open to a PR that added a "collections" tab to the home page?

I'm not sure, it feels like it would look very duplicative as all of the collections appear in the sidebar to the left? Having said that the home tab is definitely due some focus.


> Also, would you be open to a PR that allowed you to set the default tab for the home page and for collections (I'd prefer alphabetical).

Yes, open to this – in fact we already have an issue to track it: https://github.com/outline/outline/issues/1310


> The current defaults feels like it's primarily geared towards "editing".

That's certainly not supposed to to be the case, I think the homepage will be even more useful once this work lands: https://github.com/outline/outline/pull/1507 – the idea is that it's a good place to get upto speed on any meaningful changes since you last visited.

**Reply by thomseddon** [reply]
Thanks for getting back to me.

👍 on the default tabs (although I'd also love to set this globally)

On the home page, I can see what you mean - we currently use bookstack and that's how their [homepage is arranged](https://www.bookstackapp.com/#&gid=1&pid=4). That layout works well for us because less than 10% of our team would actively edit the kb, so that helps direct them quickly.

But looking at other products (trettra, slab, slite etc.) I can see they mostly adopt the "feed" approach 🤔 Albeit, either with an editable page/area or "pinned"/"favourited" items.

If you're already having thoughts about possible homepage tweaks, that's great - perhaps a good middle ground would be the ability to set a certain page as the home page? Then we could write some introductory text to ease new users in.

**Reply by alistair-a** [reply]
> the idea is that it's a good place to get upto speed on any meaningful changes since you last visited.

As a consumer of team a wiki, I don't think the recency of updates is particularly meaningful or useful. I'm normally there to answer a question. Maybe a feed of updates might be a secondary piece of information, but I suspect that only the maintainers (who in most cases should be a minority of the users) are likely to gain much utility from it.

To serve the more common case, maybe some form of directory and/or a focused search could provide a more user-centric first view.
